cumulus/parachains/runtimes/contracts/contracts-rococo/src/xcm_config.rs

```diff
@@ -227,7 +227,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = Nothing;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Everything;
+	type XcmReserveTransferFilter = Nothing;
```

Comment:
Yeah, not really relevant. I will revert this since some assets may be represented by contracts and in that case, this could possibly be a reserve location.